lib/rides/pages/search_suggestion_page.dart

```diff
@@ -248,59 +270,77 @@ class _SearchSuggestionPage extends State<SearchSuggestionPage> {
   Widget buildLocationPicker({bool isStart = true}) {
     TextEditingController controller = isStart ? _startController : _destinationController;
 
-    return Expanded(
-      child: ElevatedButton(
-        onPressed: () async {
-          final AddressSuggestion? suggestion = await showSearch<AddressSuggestion?>(
-            context: context,
-            delegate: AddressSearchDelegate(),
-            query: controller.text,
-          );
-          if (suggestion != null) {
-            controller.text = suggestion.name;
-            if (isStart) {
-              _startSuggestion = suggestion;
-            } else {
-              _destinationSuggestion = suggestion;
-            }
-            loadRides();
+    return ElevatedButton(
+      onPressed: () async {
+        final AddressSuggestion? suggestion = await showSearch<AddressSuggestion?>(
+          context: context,
+          delegate: AddressSearchDelegate(),
+          query: controller.text,
+        );
+        if (suggestion != null) {
+          controller.text = suggestion.name;
+          if (isStart) {
+            _startSuggestion = suggestion;
+          } else {
+            _destinationSuggestion = suggestion;
           }
-        },
-        child: Align(
-          alignment: Alignment.centerLeft,
-          child: Text(controller.text),
-        ),
+          loadRides();
+        }
+      },
+      child: Align(
+        alignment: Alignment.centerLeft,
+        child: Text(controller.text),
       ),
     );
   }
 
   Widget buildSearchCardList() {
```

Comment:
should we show the applied filters somewhere above this list? (ofc only if they're different from the default)

Reply:
I've also thought about that, and you have inspired me to implement that. I realized that it would be a crime to not to include a "filter" button in the row that shows you all your filters, and that at least the sorting should probably always be displayed. So, since there is now a filter button in the row and the row is always present, I removed the button from the app bar. In general, this is again a pretty big change; and for this and the whole design I would very much appreciate your feedback :)

Reply:
Oh that looks really interesting! I had just envisioned some blue text that says the filters in words, but this is pretty cool. For the rating categories, I think there should be a bit more separation between the different entries to make it more clear.
And I think the whole row should be clickable to open the filter dialog!

Reply:
Done, now there is just a little bit of ambiguity between clicking the row and dragging through it, but I don't really think that'll be a problem